### PR TITLE
fix player exile

### PR DESCRIPTION
**Pull request by tuxinal**
### Description
workaround for PlayerExileEvent

### Closes issues

- #91

### CODE REVIEW

File: src/Impostor.Server/Net/Inner/Objects/InnerMeetingHud.cs [hunk 1]
```diff
@@ -80,7 +81,15 @@ public override async ValueTask HandleRpc(ClientPlayer sender, ClientPlayer? tar
                     var tie = reader.ReadBoolean();
 
                     await _eventManager.CallAsync(new MeetingEndedEvent(_game, this));
-
+                    if(playerId != 0xFF){ // make sure a player was voted out
```
Comment (reecebenson) [inline src/Impostor.Server/Net/Inner/Objects/InnerMeetingHud.cs:84]: Linting (Spacing), please refactor to:
```csharp
if (playerId != 0xFF)
{
```

File: src/Impostor.Server/Net/Inner/Objects/InnerMeetingHud.cs [hunk 2]
```diff
@@ -80,7 +81,15 @@ public override async ValueTask HandleRpc(ClientPlayer sender, ClientPlayer? tar
...
+                        var player = _game.GameNet.GameData.GetPlayerById(playerId);
+                        if(player != null) // fixes a warning
```
Comment (reecebenson) [inline src/Impostor.Server/Net/Inner/Objects/InnerMeetingHud.cs:86]: Linting (Spacing), please refactor to:
```csharp
if (player != null)
``` Also can remove the unnecessary `// fixes a warning` comment.

File: src/Impostor.Server/Net/Inner/Objects/InnerMeetingHud.cs [hunk 3]
```diff
@@ -80,7 +81,15 @@ public override async ValueTask HandleRpc(ClientPlayer sender, ClientPlayer? tar
...
+                        {
+                            player.IsDead = true; // mark player as dead
+                            player.LastDeathReason = Api.Innersloth.DeathReason.Exile; // set death reason
+                            await _eventManager.CallAsync(new PlayerExileEvent(_game,sender,player.Controller)); // call PlayerExileEvent
```
Comment (reecebenson) [inline src/Impostor.Server/Net/Inner/Objects/InnerMeetingHud.cs:90]: No need for the comments after each of these lines, the code is self explanatory.

**Review by reecebenson — CHANGES_REQUESTED** [review]

File: src/Impostor.Server/Net/Inner/Objects/InnerMeetingHud.cs (hunk 1 above)
Comment (AeonLucid) [reply src/Impostor.Server/Net/Inner/Objects/InnerMeetingHud.cs:84]: And byte.MaxValue

File: src/Impostor.Server/Net/Inner/Objects/InnerMeetingHud.cs (hunk 3 above)
Comment (reecebenson) [inline src/Impostor.Server/Net/Inner/Objects/InnerMeetingHud.cs:90]: Please add spaces between the arguments:
```csharp
await _eventManager.CallAsync(new PlayerExileEvent(_game, sender, player.Controller));
```

**Review by reecebenson — APPROVED** [review]

**Pull request merged**